Sources/Sentry/SentryWatchdogTerminationScopeObserver.m

```diff
@@ -96,6 +152,22 @@ - (void)store:(NSData *)data
     }
 }
 
+- (void)storeContext:(NSData *)data
+{
+    @try {
+        SENTRY_LOG_DEBUG(@"Storing context data: %@", data);
+        // Override the entire file content by starting from the beginning
+        [self.fileHandleContext seekToFileOffset:0];
+        [self.fileHandleContext writeData:data];
```

Comment:
As the context doesn't change that frequently, I think we could consider storing that data on a BG thread, so we might not need using a file handle. Simple file IO operations could be enough.

Reply:
Works for me, was considering this too, thought it might be safer to have a file handle open, so we do not have to do file IO and just write data over time.

Do we need the swap files and write it twice then?